etc/notebooks/got_scotch_demo/scotch_dashboard.ipynb

```diff
@@ -354,9 +366,11 @@
     "urth_section": {}
    },
    "source": [
-    "## Radar Widget\n",
+    "### Radar Widget\n",
     "\n",
-    "Make the radar chart an ipywidget so that it updates when the selected factors change."
+    "We will call a scotch a \"factor\" for this radar visualization.\n",
```

Comment:
Ah, yes. I remember the weird naming now. The "select.factors" event is the name of a scotch which is a key for looking up the features of that scotch (the real factors). The reasoning was that it kept the widget agnostic as to the data domain vs making it a "ScotchRadarWidget". But it's still pretty terrible and damn confusing.

How about changing the variable and event to `factors_key` or `factors_about` or something?